gnat2goto/driver/tree_walk.adb

```diff
@@ -4946,8 +4940,12 @@ package body Tree_Walk is
             --  Ignored for now
               Name_No_Elaboration_Code_All |
             --  Only affects elaboration and linking so can be ignored for now
-              Name_Universal_Aliasing =>
+              Name_Universal_Aliasing |
             --  Optimisation control, should be ignored
+              Name_Preelaborable_Initialization |
```

Comment:
FWIW I don't think ignoring these is right. We _will_ need to handle these eventually.

Reply:
Well they are part of UKNI, so eventually might as well be now.

Reply:
Preelaborate is used to specify and control elaboration before calling the subprogram designated as the main program.  The conditions that allow a pragma preelaborate to be validly applied to a unit is checked by the front end.  I am hoping to use the gnat binder to define the elaboration order and again that would require no action from gnat2goto.  I think pragma Preelaborate could safely be ignored by gnat2goto.  gnat2goto may have to produce initialisation routines which facilitate elaboration but not the order in which they are called.